.github/workflows/wheels-dependencies.sh

```diff
@@ -77,67 +108,88 @@ function build {
     if [ -n "$IS_MACOS" ]; then
         build_simple xorgproto 2024.1 https://www.x.org/pub/individual/proto
         build_simple libXau 1.0.11 https://www.x.org/pub/individual/lib
+        build_simple libXdmcp 1.1.5 https://www.x.org/pub/individual/lib
```

Comment:
> Adds a build of `libXdmcp`, to avoid a dependency on the Homebrew-provided version.

I suspect you took out all the `brew remove` statements, then noticed this was being pulled from brew, and then added it here to replace that.

I'm saying that this wouldn't be part of the Pillow wheels at the moment, so if it's still not after this PR, that sounds fine to me.

This function
https://github.com/python-pillow/Pillow/blob/5bff2f3b2894ec6923c590d0c37b18177d0634bd/src/display.c#L834 is the only use of XCB in Pillow, so if it's not necessary, I don't think it needs to be here?

Reply:
My recollection was that Xdmcp was needed for XCB to compile *at all*; but I've just removed it and done a clean build, and everything worked as expected (AFAICT). I'll remove it.